backend/packageLibrary.go

```diff
@@ -95,3 +106,26 @@ func CancelInstall(pkgId string) {
 func StopApp(pkgId string) {
 
 }
+func isValidUser(users []string) (bool, error) {
```

Comment:
@bonhokage06  you dont need to request via RPC. you can system.GetDeviceInfo() to retrieve device serial. RPC still has overhead, we should only use it if really needed.